### ISSUE TITLE
data.sort not working in pie chart

### ISSUE THREAD

**Issue by jfplataroti** [naver/billboard.js#34]
## Description
Hi guys, I'm using the pie chart and I'm trying to change the order of the pie section using the property data.order, but it doesn't work. Whatever option I use I get the same result ('asc' | 'desc' | custom compare function).

this is the code example I'm using:
```javascript
bb.generate({
    bindto: "#chart",
    data: {
        columns: [
            ["data1", 30, 200, 100, 170, 150, 250],
            ["data2", 130, 100, 140, 35, 110, 50]
        ],
        type: "pie",
        order : "asc",
        colors: {
          data1: "red",
          data2: "green"
        }
    }
});
```

## Steps to check or reproduce
Here are some examples with using different sort parameters and getting always the same result:
using sort asc
 https://codepen.io/anon/pen/zzzBRO

 using sort desc
 https://codepen.io/anon/pen/BZZPVY

 using custom compare fn
 https://codepen.io/anon/pen/rwwrrE

Let me know if I'm doing something wrong or if there is a workaround for this.

thanks!

**Comment by netil** [comment]
For pie/donut types, with `data.order` can be rendered in two forms.
- default (which is sames as "desc")
- null

![image](https://user-images.githubusercontent.com/2178435/27464587-4f3b8000-5809-11e7-9731-7524e75c6c1b.png)

The order function, just works for non-arc types.

If you need data order appeared in some determined way, bind the data in that order.
If you need that pie to appeard in different angle, make some function for that as:

```js
function rotate(deg) {
    chart.internal.svg
        .style("transformOrigin", "center")
        .style("transform", "rotate("+ deg +"deg)")
        .selectAll(".bb-chart-arc text")
        .each(function() {
            var node = d3.select(this);
            node.attr("transform", node.attr("transform") + " rotate(-"+ deg +")");
        });
}
```
With that function, calling `rotate(90)` will get the result as below.
![image](https://user-images.githubusercontent.com/2178435/27464975-6dcf5792-580b-11e7-990e-941545199a9b.png)

**Comment by DigitalWheelie** [comment]
Actually, I'll extend this to say I'd love if pie chart value sorting, and/or pie chart initial rotation were built-in features/capabilities.

**Comment by netil** [comment]
fixed by `1.9.0` release.
- https://github.com/naver/billboard.js/pull/908/files#diff-7b9696c58dff42f68868edf2ea297537R30
